src/components/LicenseRevokeModal/index.jsx

```diff
@@ -47,18 +48,21 @@ class LicenseRevokeModal extends React.Component {
     const {
       user,
       sendLicenseRevoke,
+      fetchSubscriptionDetails,
+      fetchSubscriptionUsers,
+      searchQuery,
+      onRefreshError,
     } = this.props;
-
-    const options = {};
-    options.assignments = [{
-      uuid: user.uuid,
-      email: user.emailAddress,
-      licenseStatus: user.licenseStatus,
-    }];
-
+    const options = { userId: user.userId };
     return sendLicenseRevoke(options)
-      .then((response) => {
-        this.props.onSuccess(response);
+      .then(async (response) => {
+        try {
+          await fetchSubscriptionDetails();
+          await fetchSubscriptionUsers({ searchQuery });
+          this.props.onSuccess(response);
+        } catch (error) {
+          onRefreshError(error);
```

Comment:
if we failed to refresh the subscription data, let the consume of `LicenseRemindModal` know